deployment/ccip/changeset/cs_deploy_chain.go

```diff
@@ -451,16 +443,17 @@ func deployChainContracts(
 		AddedCallers: []common.Address{offRampContract.Address(), chain.DeployerKey.From},
 	})
 	if _, err := deployment.ConfirmIfNoError(chain, tx, err); err != nil {
-		e.Logger.Errorw("Failed to confirm fee quoter authorized caller update", "err", err)
+		e.Logger.Errorw("Failed to confirm fee quoter authorized caller update", "chain", chain.Name, "err", err)
 		return err
 	}
-
+	e.Logger.Infow("Added fee quoter authorized callers", "chain", chain.Name, "callers", offRampContract.Address(), chain.DeployerKey.From)
 	tx, err = nmContract.ApplyAuthorizedCallerUpdates(chain.DeployerKey, nonce_manager.AuthorizedCallersAuthorizedCallerArgs{
 		AddedCallers: []common.Address{offRampContract.Address(), onRampContract.Address()},
 	})
 	if _, err := deployment.ConfirmIfNoError(chain, tx, err); err != nil {
-		e.Logger.Errorw("Failed to update nonce manager with ramps", "err", err)
+		e.Logger.Errorw("Failed to update nonce manager with ramps", "chain", chain.Name, "err", err)
 		return err
 	}
+	e.Logger.Infow("Added nonce manager authorized callers", "chain", chain.Name, "callers", offRampContract.Address(), onRampContract.Address())
```

Comment:
doesn't the Infow one have to have the form `"string", "key", value, "key", value...`?